src/core/css/SearchResult.scss

```diff
@@ -40,6 +49,44 @@
   }
 }
 
+.SearchResult-authors {
+  display: inline-block;
+  font-weight: normal;
+  margin: 0;
+  padding: 0;
+
+  &::after {
+    content: ' — ';
+    display: inline-block;
+    margin: 0 0.25rem;
+  }
+}
+
+.SearchResult-author,
+.SearchResult-users {
+  color: $secondary-color;
+  font-size: $font-size-default;
+}
+
+.SearchResult-author {
+  border-right: 1px solid #333;
```

Comment:
There aren't any mocks for multiple authors, but that's a really good point; I just tested it with multiple authors and it's way too much info for such a small space.

Even "Mark Striemer and 2 others" is quite long in there, so I think we should go with either the first author in the list or "3 authors".

I'll go with just the first author's name for now and we can show more in the details page. Thanks for pointing that out.